monitor.py

```diff
@@ -21,6 +22,8 @@ def __init__(self, config):
     def on_created(self, event):
         send_message(self.token, self.chat_id)
         print("Sending message...")
+        # Uncoment the following line to enable windows notifications
```

Comment:
There might be a more elegant way to enable windows notifications. I am currently working on mac support so I was thinking maybe the option to enable/disable windows notifications should be in the config
```suggestion
        # Uncomment the following line to enable windows notifications
```